ckanext/canada/templates/internal/base.html

```diff
@@ -13,4 +13,6 @@
 
 {%- block adobe_analytics -%}{%- endblock -%}
 
-{%- block adobe_analytics_meta -%}{%- endblock -%}
\ No newline at end of file
+{%- block adobe_analytics_meta -%}{%- endblock -%}
```

Comment:
why are we adding forsee to the registry?

Reply:
oh I get it, you're hiding the block with this

Reply:
no longer need to have an empty block for registry